src/Storage/RevisionableStorageInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\entity\Storage\RevisionableStorageInterface.
+ */
+
+namespace Drupal\entity\Storage;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+
+/**
+ * Defines some additional storage methods used for revision support.
+ */
+interface RevisionableStorageInterface extends EntityStorageInterface {
+
+  /**
+   * Counts the number of revisions in the default language.
+   *
+   * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+   *   The entity.
+   *
+   * @return int
+   *   The number of revisions in the default language.
+   */
+  public function countDefaultLanguageRevisions(ContentEntityInterface $entity);
+
+  /**
+   * Gets a list of revision IDs for a specific entity.
+   *
+   * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+   *   The content entity.
+   *
+   * @return int[]
+   *   Entity revision IDs (in ascending order).
+   */
+  public function revisionIds(ContentEntityInterface $entity);
```

Comment:
I don't know why node has this method. This can easily be done with an entity query?